packages/rest-crud/src/crud-rest.controller.ts

```diff
@@ -70,11 +70,11 @@ export interface CrudRestController<
    */
   readonly repository: EntityCrudRepository<T, IdType>;
 
-  /**
-   * Implementation of the endpoint `POST /`.
-   * @param data Model data
-   */
-  create(data: Omit<T, IdName>): Promise<T>;
+  // /**
+  //  * Implementation of the endpoint `POST /`.
+  //  * @param data Model data
+  //  */
+  // create(data: Omit<T, IdName>): Promise<T>;
```

Comment:
cc @bajtos Since the type of `data` is not determined (w/ or w/o id), I removed the create function in this interface. Thought?

Reply:
I am quite unhappy with the changes made in this file, I feel they are introducing too much complexity. We have started with one option - id is required or generated - and have two controller classes. Imagine what is going happen when the number of options grows. Are we going to have 8 different classes when the number of options grows to 3?

I prefer to sacrifice compile-time safety and keep the implementation simple. I expect there will be very few people consuming these runtime-generated controllers directly from TypeScript code.

I tried my suggestions to replace `Omit<T, IdName>` with `Todo & {id?: ID}` and I see it does not work. Can we use the following type instead? 

```ts
   async create(
      data: T | Omit<T, IdName>,
    ): Promise<T>;
```